src/gui/qgsmaptoolcapture.h

```diff
@@ -137,6 +132,12 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
      */
     void closePolygon();
 
+  protected slots:
+    /**
+     * Stop capturing
+     */
+    void stopCapturing();
```

Comment:
`stopCapturing()` is already protected. The change is to make it a slot. Why would that break things?